src/Control Tasks/EEPROMControlTask.cpp

```diff
@@ -22,7 +22,7 @@ void EEPROMControlTask::save_wait_time()
 #ifdef VERBOSE
         Serial.println("Current wait time value: " + String(wait_time));
 #endif
-        if (wait_time > sfr::eeprom::alloted_time) {
+        if (wait_time >= sfr::eeprom::alloted_time) {
```

Comment:
I would change the comment for this to be "reaches the wait time" instead of "exceeds the wait time" since its now >=